requirements.txt

```diff
@@ -26,3 +26,4 @@ tweepy >=4.0.0
 ufal.udpipe >=1.2.0.3
 wikipedia
 yake
+sentence_transformers
```

Comment:
I know it is not ready yet. I am just adding the note that we do not forget: sentence_transformers should be removed requirements.

Reply:
done